plugins/serializers/registry.go

```diff
@@ -188,8 +191,8 @@ func NewWavefrontSerializer(prefix string, useStrict bool, sourceOverride []stri
 	return wavefront.NewSerializer(prefix, useStrict, sourceOverride)
 }
 
-func NewJSONSerializer(timestampUnits time.Duration) (Serializer, error) {
-	return json.NewSerializer(timestampUnits)
+func NewJsonSerializer(timestampUnits time.Duration, timestampFormat string) (Serializer, error) {
```

Comment:
Can you undo this capitalization change to appease the linter?
```suggestion
func NewJSONSerializer(timestampUnits time.Duration, timestampFormat string) (Serializer, error) {
```

Reply:
Ah, thanks, I was too stupid to grasp the idea of the linter.